README.md

```diff
@@ -1,69 +1,81 @@
 # Flamboi Slack API
+
 Flameboi is CodeDevils's Slack API bot that posts interactive messages and offers commands to users on CodeDevils' Slack.
 
-# How to Install
+## How to Install
+
+### 1) (Optional) Install Virtualenv
 
-## 1) (Optional) Install Virtualenv
 This is an optional step, but is reccomended to keep your local Python dependency versions separate from Flamboi's. With many Python projects, bugs can exist in apps when versions of the same dependencies are shared globally across your code repos.
 
-### Linux
-#### Install package
+#### Linux
+
+##### Install package
+
 `sudo apt-get install python-virtualenv`
-#### Activate easy installation scirpt
+
+##### Activate easy installation scirpt
+
 `sudo easy_install virtualenv`
-#### Install pip Virtualenv
+
+##### Install pip Virtualenv
+
 `sudo pip install virtualenv`
-#### Ensure Virtualenv is Installed
+
+##### Ensure Virtualenv is Installed
+
 Ensure Virtualenv is installed by calling `virtualenv`
 
 > If virtualenv appears to not be installed, ensure that it is on your path. You can do this by adding the installation foler to the PATH variable within your profile. An example would be within ~/.bash_profile or ~/.bashrc, adding `PATH=$PATH:/home/pi/.local/bin`.
```

Comment:
Not really important... `/home/pi/` instead of `~` has returned from two commits ago

Reply:
lol good catch, fixed by pulling the recent merge